app/dashboard/migrations/0031_auto_20180105_1659.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2018-01-05 23:59
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0030_auto_20171228_1431'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='bounty',
+            name='github_comments',
+            field=models.IntegerField(default=0),
+        ),
+        migrations.AlterField(
```

Comment:
added help_text to 2 of the fields -- keeping them here to avoid generating extra migrations